onnx/test/optimizer_test.py

```diff
@@ -766,42 +776,48 @@ def test_fuse_matmul_add_bias_into_gemm_multiple_use_no_fuse(self):  # type: ()
     def test_fuse_pad_into_conv(self):  # type: () -> None
```

Comment:
Is it possible to keep the test for old Pad?

Reply:
I tried. But I am not sure we can create a node for an older opset. I notive that there is something within the `circleci` CI that covers the old optimizer logic because when I removed the old logic, it broke the CI and when the optimizer supported both versions - the CI passed....

Reply:
Ok. I think old optimization is implicitly tested somewhere. Could you please one an issue to discuss if we should add more tests for old optimizations? Then, we can close this comment.

Reply:
sure, will do.

Reply:
Raised this issue here - https://github.com/onnx/onnx/issues/2313. Resolving comment as prescribed. Thanks!

Reply:
I preserved the old Pad tests. Added some infrastructure code for it. Will resolve the issue after PR merge.